src/Components/Inquiry/config.ts

```diff
@@ -26,6 +28,26 @@ interface UseEngine {
 export const useEngine = ({ context, onDone }: UseEngine) => {
   const visited = useRef(new Visited("inquiry"))
 
+  const conditions = {
+    askSpecialist: () => {
+      return !!context.current?.askSpecialist
+    },
+    hasBasicInfo: () => {
+      return (
+        !!context.current?.profession &&
+        !!context.current?.location?.city &&
+        !!context.current?.otherRelevantPositions &&
+        !!context.current?.shareFollows
```

Comment:
I'm surprised to see a check for `shareFollows` included here because the profile prompt only lets users update the other 3 fields. Why did you include it?

Reply:
(I'm also not sure if `otherRelevantPositions` should be included. I certainly don't have any to add to my user profile.)

Reply:
Yeah this is just the logic that's currently in production. You're right though.